### DISCUSSION TITLE
请问  xmake是否支持VCPKG的包

### DISCUSSION THREAD

**Discussion by Super1Windcloud** [xmake-io/xmake#5846] [General]
###  请问  xmake是否支持VCPKG的包, VCPKG数量是多于xmake的,  如果 xmake仓库没有的包,VCPKG有, 那么这种情况的是调用VCPKG去安装然后链接吗 ?

**Comment by star-hengxing** [comment]
https://xmake.io/#/package/remote_package?id=add-vcpkg-dependency-package

不过一般只有官方包仓库才能获得最佳体验，可以来提 feature request 请求加入包，或者自己动手打包 pr

**Reply by waruqi** [reply]
vcpkg/conan/apt/pacman/homebrew/dub/cargo 等各大仓库的包，都支持，你随意偷。文档和 readme 里面都有详细说明，仔细看
